### DISCUSSION TITLE
Low Priority Request: Followers Listing on website and ingame.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17099] [Ideas] [answered]
This is an idea I've wanted in the game for a LONG time. I would like to see a menu in the F9 playerlist to filter for followers similar to filtering for friends. My main reasoning behind this request is I am a multiplayer main and enjoy hosting my own lobbies but finding players to join can be a hassle especially when you don't mutual many people. I'd like to be able to invite people from my followers list as well as just be able to see who all is following me.

I've made this a Low Priority or Priority 3 request due to the fact that I don't know if this data is stored per user in the first place and if it isn't it could add a LOT of overhead trying to gather all the people that follow you especially for streamers and people that run bots such as myself as even I a low 6 digit have over 200 followers now.

**Comment by frenzibyte** [comment, marked as answer]
Tracked in https://github.com/ppy/osu-web/issues/5227.
